### ISSUE TITLE
Buggy creation of config.cpp

### ISSUE THREAD

**Issue by Krzmbrzl** [X39/ObjectOrientedScripting#36]
When just using an entryPoint-Function outside of a class/namespace the config.cpp misses a category/Tag and this results in ArmA not executing the function at all!

Code causing this issue:

```
static void postInit() {
    SQF hint("Hallo ArmA!");
}
```

**Comment by Krzmbrzl** [comment]
Btw. the gernerated config.cpp looks like this:

```
class cfgFunctions
{
    class Generic
    {
        class postInit
        {
            file = "postInit.sqf";
            preInit = 0;
            postInit = 1;
            preStart = 0;
            recompile = 0;
            ext = ".sqf";
            headerType = 0;
        };
    };
};
```

**Comment by X39** [comment]
solved in 0.6.0
